.gitignore

```diff
@@ -99,3 +99,6 @@ ENV/
 
 # mypy
 .mypy_cache/
+
+#Editor config
```

Comment:
Please ignore me, if you think I am being pedantic. Your contribution is appreciated.

Missing space after `#`.

Reply:
Done. :)